src/textual/reactive.py

```diff
@@ -169,53 +159,41 @@ def _reset_object(cls, obj: object) -> None:
         getattr(obj, "__watchers", {}).clear()
         getattr(obj, "__computes", []).clear()
 
-    def __set_name__(self, owner: Type[MessageTarget], name: str) -> None:
-
+    def __set_name__(self, owner: Reactable, name: str) -> None:
```

Comment:
`Reactable` rather than `Type[MessageTarget]`. Probably, again, older code?

Reply:
The rest of the descriptor uses Reactable, which supports the necessary interface for reactives. Although I think owner is the class and not an instance. So it should be `Type[Reactable]`

Reply:
Correct, I'll fix.